.github/workflows/build.yml

```diff
@@ -168,6 +168,9 @@ jobs:
           jupyter lite status --debug || exit 1
       - name: Setup pip (test)
         run: ${{ matrix.python-command }} -m pip install -r requirements-test.txt
+      - name: Pin pytest for pypy
+        if: ${{ matrix.python-version == 'pypy-3.8' }}
+        run: ${{ matrix.python-command }} -m pip install "pytest<8"
```

Comment:
Could write something (e.g. link to issue) on `$GITHUB_STEP_SUMMARY`...

```
echo 'See https://github.com/jupyterlite/jupyterlite/issues/1308' >> $GITHUB_STEP_SUMMARY
```

Reply:
Sure let's try that.

Reply:
Done in https://github.com/jupyterlite/jupyterlite/pull/1309/commits/6a6dbe36e5a8425e6af27e0fe4826a0a1c32f2a5